core/src/main/java/org/frankframework/mongodb/MongoDbSender.java

```diff
@@ -167,12 +166,12 @@ public void close() throws SenderException {
 	@Override
 	public SenderResult sendMessage(Message message, PipeLineSession session) throws SenderException, TimeoutException {
 		message.closeOnCloseOf(session, this);
-		MongoAction mngaction = getAction();
-		try (MessageOutputStream target = mngaction==MongoAction.FINDONE || mngaction==MongoAction.FINDMANY ? MessageOutputStream.getTargetStream(this, session, null) : new MessageOutputStreamCap(this, null)) {
+		MongoAction mongoAction = getAction();
+		try (MessageOutputStream target = mongoAction==MongoAction.FINDONE || mongoAction==MongoAction.FINDMANY ? MessageOutputStream.getTargetStream(this, session, null) : new MessageOutputStreamCap(this, null)) {
```

Comment:
Is het mogelijk om deze check op findone/findmany in de enum op te nemen om dit wat leesbaarder te maken? Bv `mongoAction.isFindAction` oid?

Reply:
Will do. Maar je hebt dit na de merge neergezet, dus ik pak dit op in een volgende PR. Goede suggestie wel 👍🏻